app/views/servers/show.html.erb

```diff
@@ -9,6 +9,11 @@
   <%= @server.url %>
 </p>
 
+<p>
+  <strong>Kind:</strong>
```

Comment:
The naming here would be good to make consistent ("Kind" here and "Backend" elsewhere). How about calling it "Type" or "Server type" everywhere"?